atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/keyvalue/impl/TableRemappingKeyValueService.java

```diff
@@ -156,6 +161,10 @@ private Set<TableReference> getShortTableReferencesForExistingTables(Set<TableRe
                 tableNames.add(tableMapper.getMappedTableName(tableRef));
             } catch (TableMappingNotFoundException e) {
                 // Table does not exist - do nothing
+                log.debug(
+                        "Could not find short table reference for an existing table",
+                        LoggingArgs.tableRef(tableRef),
+                        e);
```

Comment:
Since we didn't wrap this around Safe...Exception(e), this should be unsafe by default